### ISSUE TITLE
Remove Custom Error Handler

### ISSUE THREAD

**Issue by clarkepeterf** [project-lux/lux-marklogic#317]
**Problem Description:**
ML has a custom error handler which is not necessary and could contribute to v8 crashes

It was determined as part of https://github.com/project-lux/lux-marklogic/issues/89 that we should get rid of the error handler

**Expected Behavior/Solution:**
Remove the custom error handler

**Requirements:**
- [ ] Delete customErrorHandler.mjs
- [ ] Change the value of the error-handler property in both app server configuration files. "error-handler": "" may work.
- [ ] Otherwise, I believe the value is /MarkLogic/rest-api/error-handler.xqy.
- [ ] Delete TRACE_NAME_ERROR and its export.
- [ ] Delete the LuxError documentation in lux-backend-deployment.md
- [ ] Close https://github.com/project-lux/lux-marklogic/issues/178.

**Needed for promotion:**
~~If an item on the list is not needed, it should be crossed off but not removed.~~

~~- [ ] Wireframe/Mockup - Mike~~
~~- [ ] Committee discussions - Sarah~~
~~- [ ] Feasibility/Team discussion - Sarah~~
~~- [ ] Backend requirements - TBD~~
~~- [ ] Frontend requirements- TBD~~
~~- [ ] Are new regression tests required for QA - Amy~~
~~- [ ] Questions~~
~~- List of questions for discussions. Answers should be documented within the issue.~~

**UAT/LUX Examples:**

- Not testable via frontend. @clarkepeterf  can share proof that this is done.

~~**Dependencies/Blocks:**~~

~~- Blocked By: Issues that are blocking the completion of the current issue.~~
~~- Blocking: Issues being blocked by the completion of the current issue.~~

**Related Github Issues:**
- https://github.com/project-lux/lux-marklogic/issues/89

~~**Related links:**~~

~~- These links can consist of resources, bugherds, etc.~~

~~**Wireframe/Mockup:**~~
~~Place wireframe/mockup for the proposed solution at end of ticket.~~

**Comment by roamye** [comment]
from dev mtg (12/16) this issue is to be unassigned from the milestone until it is ready to be worked on.

**Comment by zachjschwartz** [comment]
@brent-hartwig can you confirm this is good to close?

**Comment by brent-hartwig** [comment]
@zachjschwartz, yes this may be closed.  The custom error handler was removed in release3.0.

**Comment by zachjschwartz** [comment]
UAT 3/19: Keep open until next Blue/Green switch.

**Comment by zachjschwartz** [comment]
@brent-hartwig is this issue deployed and clear to close?

**Comment by brent-hartwig** [comment]
@zachjschwartz.  It is part of Backend v3.0 ([changelog](https://github.com/project-lux/lux-marklogic/blob/main/CHANGELOG.md)) and thus being held up alongside #638, #639, and #643 per previous discussion/decision.

**Comment by zachjschwartz** [comment]
@brent-hartwig are this issue and 638, 639 and 643 deployed and clear to close?

**Comment by brent-hartwig** [comment]
Yes, @zachjschwartz
